packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -9,22 +9,12 @@ import type { TypedDocumentNode } from '@urql/vue'
 import type { FragmentDefinitionNode } from 'graphql'
 import { print } from 'graphql'
 import { testUrqlClient } from './clientTestUrqlClient'
-import type { MutationResolverCallback as MutationResolver, ResultType } from './clientTestUrqlClient'
+import type { MutationResolverCallback as MutationResolver } from './clientTestUrqlClient'
 import type { Component } from 'vue'
 import { computed, watch, defineComponent, h, toRaw } from 'vue'
 import { each } from 'lodash'
 import { createI18n } from '@cy/i18n'
-
-/**
- * This variable is mimicing ipc provided by electron.
- * It has to be loaded run before initializing GraphQL
- * because graphql uses it.
- */
-
-(window as any).ipc = {
-  on: () => {},
-  send: () => {},
-}
```

Comment:
where'd this go?

Reply:
It's been gone in the 10.0 branch since like August, got replaced with the local express server